main.py

```diff
@@ -44,14 +48,25 @@ def cbz_folder(folder_name:str):
 
     # create cbz of format "folder_name.cbz" and contents of `folder`
     # -t means no cbz is created
-    created_cbz_name = str(os.path.join(os.getcwd(), f"{folder_name}.cbz"))
-    VERBOSITY_PRINT(1, f"Creating {created_cbz_name}")
-    if(not isTest):
-        zip_name = shutil.make_archive(folder_name, 'zip', os.path.join(os.getcwd(), folder_name))
-        p = Path(zip_name)
-        p.rename(p.with_suffix(".cbz"))
+    if(not isMerge):
+        created_cbz_name = str(os.path.join(os.getcwd(), f"{folder_name}.cbz"))
+        VERBOSITY_PRINT(1, f"Creating {created_cbz_name}")
+        if(not isTest):
+           zip_name = shutil.make_archive(folder_name, 'zip', os.path.join(os.getcwd(), folder_name))
+           p = Path(zip_name)
+           p.rename(p.with_suffix(".cbz"))
+           NUMBER_ZIPPED += 1
+    else: # executed when chapters are to be merged, instead of zipping one directory at a time, the directory contents are copied to a working directory to be zipped at the end
+        if(not isTest):
+            # because filenames will likely collide if we copy the names as they exist in the source dir, the files are named sequentially starting from 0
+           files=os.listdir(os.path.join(os.getcwd(),folder_name))
+           for fname in files:
+            file_extension = Path(os.path.join(os.getcwd(),folder_name,fname)).suffix
+            shutil.copy(os.path.join(os.path.join(os.getcwd(),folder_name,fname)),os.path.join(os.getcwd(),"working_merge/",(str(NUMBER_FILES_MERGED)+file_extension)))
```

Comment:
I think something is wrong with the destination path here. I see `tachiyomi_path/working_merge/` was created on line 126, so is the second argument of `shutil.copy(...)` supposed to be `os.path.join(tachiyomi_path,"working_merge",(str(NUMBER_FILES_MERGED)+file_extension))`?



Without the change I got the following error in my little toy folder 
```
Traceback (most recent call last):
  File "main.py", line 128, in <module>
    cbz_search(tachiyomi_path, directory_list)
  File "main.py", line 107, in cbz_search
    cbz_search(folder_name, directory_only_list)
  File "main.py", line 107, in cbz_search
    cbz_search(folder_name, directory_only_list)
  File "main.py", line 107, in cbz_search
    cbz_search(folder_name, directory_only_list)
  File "main.py", line 111, in cbz_search
    cbz_folder(folder_name)
  File "main.py", line 65, in cbz_folder
    shutil.copy(os.path.join(os.path.join(os.getcwd(),folder_name,fname)),os.path.join(os.getcwd(),"working_merge/",(str(NUMBER_FILES_MERGED)+file_extension)))  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.8_3.8.2800.0_x64__qbz5n2kfra8p0\lib\shutil.py", line 418, in copy
    copyfile(src, dst, follow_symlinks=follow_symlinks)
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.8_3.8.2800.0_x64__qbz5n2kfra8p0\lib\shutil.py", line 264, in copyfile
    with open(src, 'rb') as fsrc, open(dst, 'wb') as fdst:
FileNotFoundError: [Errno 2] No such file or directory: 'D:\\test\\tachiyomi\\downloads\\MangaDex\\Berserk\\working_merge/0.jpg'
```